pkg/drivers/qemu/qemu.go

```diff
@@ -445,7 +445,9 @@ func (d *Driver) Start() error {
 			d.diskPath())
 	}
 
-	if stdout, stderr, err := cmdOutErr(d.Program, startCmd...); err != nil {
+	socketCmd := append([]string{"/var/run/socket_vmnet", d.Program}, startCmd...)
+	fmt.Printf("socketCmd: %v\n", socketCmd)
+	if stdout, stderr, err := cmdOutErr("/opt/socket_vmnet/bin/socket_vmnet_client", socketCmd...); err != nil {
```

Comment:
The paths should be probably configurable.
Also needs documentation.

Reply:
That's a good idea, I was hoping to get a minimum viable product working. I'll see if I can make a flag for allowing for a custom `socket_vmnet_client` and `socket_vmnet` path.

I appreciate your review.

Reply:
Updated PR with customizable flag paths for socket_vmnet binaries, please let me know if any more changes are needed.